src/CoursesPage.php

```diff
@@ -76,13 +76,13 @@ public function getCoursesByCategories(array $category_ids): array {
   /**
    * Returns list of Courses page entities by given Course.
    *
-   * @param \Drupal\node\NodeInterface $course
+   * @param \Drupal\node\NodeInterface|null $course
    *   Course entity.
    *
-   * @return array|\Drupal\node\NodeInterface[]
+   * @return \Drupal\node\NodeInterface|array
```

Comment:
This change is not correct - we return either array of NodeInterfaces or an array.

Reply:
To be honest, no idea why I changed it. Will revert.